_stbt/core.py

```diff
@@ -687,8 +687,11 @@ def raise_in_user_thread(exception):
         display[0].tell_user_thread(exception)
     mainloop = _mainloop()
 
-    sink_pipeline = SinkPipeline(
-        gst_sink_pipeline, raise_in_user_thread, save_video)
+    if not gst_sink_pipeline and not save_video:
+        sink_pipeline = NoSinkPipeline()
```

Comment:
Oh yeah, it's not `"disable"` it's `""`. How silly of me. `""` is better because it's not a valid GStreamer pipeline description.